src/modifiers/ContentSecurityPolicyModifier.py

```diff
@@ -17,14 +17,17 @@ def _mv2(self):
     pass
 
   def _mv3(self):
+    candidate = {}
     manifest = self.wrapper.manifest
     key = 'content_security_policy'
-    if key not in manifest: return
-    Logger().log("Changing content_security_policy in manifest.json")
-    value = manifest[key]
-    candidate = {
-      "extension_pages": value,
-      "sandbox": ""
-    }
+    if key in manifest:
+      value = manifest[key]
+      candidate["extension_pages"] = value
+    if "sandbox" in manifest and key in manifest["sandbox"]:
+      candidate["sandbox"] = manifest["sandbox"]
```

Comment:
```suggestion
      candidate["sandbox"] = manifest["sandbox"][key]
```

I may be missing something, but it looks like the current impl is assigning the entire `sandbox` object from the original manifest to the new manifest's `content_security_policy.sandbox`. Also my Python is weak, so I'm not 100% sure if this is the right access pattern.

Reply:
Good catch.

Reply:
Good catch!